dsa-solutions/lc-solutions/0600-0699/0618-students-report-by-geography.md

```diff
@@ -0,0 +1,112 @@
+---
+id: students-report-by-geography
+title: Students Report By Geography
+sidebar_label: 0618-students-report-by-geography
+tags:
+- Database
+description: "Given a table that indicates the name of a student and the continent they came from if it is unknown which continent has the most students, could you write a solution to generate the student report?"
+---
+
+## Problem
+Table: `Student`
+
+```
++-------------+---------+
+| Column Name | Type    |
++-------------+---------+
+| name        | varchar |
+| continent   | varchar |
++-------------+---------+
+
+```
+This table may contain duplicate rows.
+Each row of this table indicates the name of a student and the continent they came from.
+
+
+
+A school has students from Asia, Europe, and America.
+Write a solution to <a href="https://en.wikipedia.org/wiki/Pivot_table" target="_blank">pivot</a> the continent column in the `Student` table so that each name is **sorted alphabetically** and displayed underneath its corresponding continent. The output headers should be `America`, `Asia`, and `Europe`, respectively.
```

Comment:
replace 
`<a href="https://en.wikipedia.org/wiki/Pivot_table" target="_blank">pivot</a>` 
to 
`[pivot](https://en.wikipedia.org/wiki/Pivot_table)`